test/database/download_test.go

```diff
@@ -0,0 +1,29 @@
+package database
+
+import (
+	"context"
+	"testing"
+
+	"github.com/SkynetLabs/skynet-accounts/database"
+	"github.com/SkynetLabs/skynet-accounts/test"
+)
+
+// TestDownloadCreateAnon ensures that UploadCreate can create anonymous downloads.
+func TestDownloadCreateAnon(t *testing.T) {
+	ctx := context.Background()
```

Comment:
`nit` this is not great in tests because it'll just hang if something is wrong on db level

Reply:
I'm sure I follow. What is going to hang, the test or the context?
The context should be garbage collected after the function exits, right?

Reply:
the test might hang, because the context won't time out 🤷‍♂️    
was just a suggestion to add a timeout